src/Cli/func/ExtensionBundle/ExtensionBundleHelper.cs

```diff
@@ -59,6 +60,8 @@ public static async Task GetExtensionBundle()
             }
             catch (Exception)
             {
+                ColoredConsole.WriteLine("Failed to download the Extension Bundle. Please check your network connection and try again.");
```

Comment:
Should we wrap the network call in a retry mechanism so that a transient network issue is not throwing the error?